packages/activation-notice/src/app/components/link.js

```diff
@@ -27,7 +27,7 @@ const Link = styled.a`
   color: ${({ theme }) => theme.colors.link.fg};
   text-decoration: none;
   cursor: pointer;
-
+  display: block;
```

Comment:
Curious, why was this needed? Does this cause any visual changes?

Reply:
I removed the br in thing strings. Display block means it goes to a new line

Reply:
So I take it that `createInterpolateElement` doesn't support `br` then? Would be good to note that in the PR description.